pkg/rpctype/rpctype.go

```diff
@@ -17,12 +17,23 @@ import (
 // ExecutionRequest describes the task of executing a particular program.
 // Corresponds to Fuzzer.Request.
 type ExecutionRequest struct {
-	ID               int64
-	ProgData         []byte
-	ExecOpts         ipc.ExecOpts
-	NewSignal        bool
+	ID        int64
+	ProgData  []byte
+	ExecOpts  ipc.ExecOpts
+	NewSignal bool
+	// If set, ProgData contains compiled executable binary
+	// that needs to be written to disk and executed.
+	IsBinary bool
+	// If set, fully reset executor state befor executing the test.
+	ResetState bool
```

Comment:
With `ResetState` in `ExecutionRequest`, we no longer need to pass the `flagResetAccState` to `syz-fuzzer`. We can just set it on the host side.

Reply:
Done. I like removing more code from the fuzzer.

Reply:
There are still some parts left (e.g. `flagResetAccState` in syz-fuzzer).

Reply:
Done. PTAL